src/content/en/fundamentals/performance/rail.md

```diff
@@ -36,11 +37,15 @@ Understand how users perceive performance delays:
   <tbody>
     <tr>
       <td data-th="Delay">0 - 16ms</td>
```

Comment:
Note that this section is talking about user perception, so I left the reference to 16ms here. However, I clarify below that this number includes browser work time.